pom.xml

```diff
@@ -153,7 +152,42 @@
 			</plugin>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>${checkstyle.version}</version>
+				<executions>
+					<execution>
+						<id>verify-style</id>
+						<phase>process-classes</phase>
+						<goals>
+							<goal>check</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<logViolationsToConsole>true</logViolationsToConsole>
+					<checkstyleRules>
+						<module name="Checker">
+							<!-- Checks for Size Violations.                    -->
+							<!-- See http://checkstyle.sf.net/config_sizes.html -->
+							<module name="FileLength">
+								<property name="max" value="500" />
```

Comment:
I was thinking along the lines of 200. What do you think?

Reply:
I don't have any opinion I entered 500 because it doesn't violate the existing code. I didn't want to change files without discussing them.

Reply:
Can you please confirm which files currently exceed 200 lines?